tests/test_grpc.py

```diff
@@ -93,6 +97,24 @@ def test_download_missing_file(mapdl, tmpdir):
         mapdl.download('__notafile__', target)
 
 
+@skip_launch_mapdl  # need to be able to start/stop an instance of MAPDL
+def test_grpc_custom_ip():
+    from ansys.mapdl.core import launch_mapdl
+    import re
+    
+    ip = '127.0.0.2'
+    mapdl = launch_mapdl(ip=ip)
+
+    output_file = mapdl._download_as_raw('.__tmp__.out')
+    output = output_file.decode().splitlines()
+
+    ip_in_output = re.findall('Server listening on : (.*):', output)
+    
+    mapdl.exit()
+
+    assert ip == ip_in_output
```

Comment:
This should fail.  Please test this locally since we can't test this on our CI/CD.

For this to work, you'll need `ip_in_output[0]` because `re.findall` returns a list.

Reply:
Fixed.